homeassistant/components/aosmith/config_flow.py

```diff
@@ -0,0 +1,110 @@
+"""Config flow for A. O. Smith integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+from py_aosmith import AOSmithAPIClient, AOSmithInvalidCredentialsException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import aiohttp_client
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("email"): str,
```

Comment:
```suggestion
        vol.Required(CONF_EMAIL): str,
```
please use constants. (Not sure if this one exists but I'm guessing so)